python/xorbits/_mars/_utils.pyx

```diff
@@ -142,6 +142,10 @@ cdef inline list iterative_tokenize(object ob):
 cdef inline tuple tokenize_numpy(ob):
     cdef int offset
 
+    # Return random bytes and metadata for objects > 64MB
+    if ob.nbytes > 64 * 1024 ** 2:
+        return os.urandom(16), ob.dtype, ob.shape, ob.strides
```

Comment:
To my understanding, this function should return a hash of given object, which means for the same object, this function should always return a same binary. 

It seems that your changes breaks the principle?

Reply:
The purpose of this change is to deal with `mmh_hash_bytes(ob.ravel().view('i1').data)` will raise `segmentation fault` when the size of `ob` is very large (because the `mmh_hash_bytes` will use all the entry of 'ob' to calculate the hash value). In this case, I think there are three possible solution:
1. Use a portion of ‘ob’ to calculate the hash value (suffer from the randomness)
2. Use a mapping function to map `ob` to a smaller size object (suffer from the randomness and choosing the mapping)
3. Just random a 16 bytes hash value. (suffer from the case you mention)

#After discussing with @aresnow1 and @qianduoduo0904, we choose to use method 3 (for super big data, ignore the sameness of hash value)